src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/config/ServiceConfigItem.java

```diff
@@ -62,5 +74,26 @@ public void setAllowCreatingCaseBeforePaymentsAreProcessed(boolean allowCreating
         this.allowCreatingCaseBeforePaymentsAreProcessed = allowCreatingCaseBeforePaymentsAreProcessed;
     }
 
+    public String getSurnameMapping(String formType) {
+        return surnameMappings.get(formType);
+    }
+
+    public void setSurnameMappings(List<FormFieldMapping> surnameMappings) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Found 'UR'-anomaly for variable 'e' (lines '81'-'88').](https://app.codacy.com/app/HMCTS/bulk-scan-orchestrator/pullRequest?prid=4498197)